TarkovBot.py

```diff
@@ -0,0 +1,68 @@
+import discord
+from config_bot import TOKEN
+import pandas as pd
+from discord.ext import commands
+pd.options.mode.chained_assignment = None
+
+intents = discord.Intents().all()
+bot = commands.Bot(command_prefix='!',intents=intents)
+client = discord.Client(intents=intents)
+station_keyword = ['Workbench', 'Medstation', 'Nutrition', 'Lavatory']
+
+
+def data_import():
+    pass
+
+# command to clear channel messages
+@bot.command(name='clear')
+async def clear(ctx, amount=5):
+    await ctx.channel.purge(limit=amount)
+    await ctx.send("Messages have been cleared")
+
+
+# command for showing tarkov crafts.
+@bot.command(name='crafts', help=f"args:{','.join(station_keyword)}")
+async def give_stats(ctx, arg):
+    # sorts the data by profit per hour index
+    df = pd.read_csv("Data/TarkovData.csv")
+    df = df[df['profit_per_hour'] > 0]
+    if arg == 'help':
```

Comment:
My recommendation is to use blank lines before statements like `if`, `for`, `while` etc.